packages/app/src/modules/options.js

```diff
@@ -200,26 +155,32 @@ export const getOptionsForRequest = () => {
     )
 }
 
-const isNotDefault = (optionsFromVisualization, prop) => {
-    return Boolean(
-        optionsFromVisualization[prop] &&
-            optionsFromVisualization[prop] !== options[prop].defaultValue
-    )
-}
-
 export const getOptionsFromVisualization = visualization => {
     const optionsFromVisualization = {
         ...getOptionsForUi(),
         ...pick(visualization, Object.keys(options)),
     }
 
-    optionsFromVisualization.baseLine =
-        isNotDefault(optionsFromVisualization, 'baseLineLabel') ||
-        isNotDefault(optionsFromVisualization, 'baseLineValue')
-
-    optionsFromVisualization.targetLine =
-        isNotDefault(optionsFromVisualization, 'targetLineLabel') ||
-        isNotDefault(optionsFromVisualization, 'targetLineValue')
+    optionsFromVisualization.axes = optionsFromVisualization.axes?.map(axis => {
```

Comment:
Perhaps the `?` is redundant as we discussed.
(Also it was the first time I saw it used to check the existence of a function of a primitive type, so I wasn't sure its use in this context is legit)
```suggestion
    optionsFromVisualization.axes = optionsFromVisualization.axes.map(axis => {
```